### PR TITLE
Fix #241 epicgames.com

### LINKED ISSUE #241: www.epicgames.com
@adguard-bot commented on [Mon Jul 26 2021](https://github.com/AdguardTeam/AdguardFilters/issues/89202)

### Issue URL (Incorrect Blocking)
 [https://www.epicgames.com/id/login?productName=epic-games&lang=ru&redirectUrl=https%3A%2F%2Fwww.epicgames.com%2Fsite%2Fru%2Fhome&client_id=5a6fcd3b82e04f8fa0065253835c5221&noHostRedirect=true](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.epicgames.com%2Fid%2Flogin%3FproductName%3Depic-games%26lang%3Dru%26redirectUrl%3Dhttps%253A%252F%252Fwww.epicgames.com%252Fsite%252Fru%252Fhome%26client_id%3D5a6fcd3b82e04f8fa0065253835c5221%26noHostRedirect%3Dtrue)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.6.6
Browser: | YaBrowser
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base
Other extensions used: | Встроенный блокировщик рекламы YAbrowser; расширение SaveFromNet

---

@BlazDT commented on [Mon Jul 26 2021](https://github.com/AdguardTeam/AdguardFilters/issues/89202#issuecomment-886724621)

Screenshot not matching used product. Please check Popup-Blocker @tvinzz

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/PopupBlocker/issues/241

### CODE REVIEW

File: compiler/MetadataUtils.ts [hunk 1]
```diff
@@ -364,6 +364,9 @@ export default class MetadataUtils {
         // https://github.com/AdguardTeam/PopupBlocker/issues/238
         '*://online.bfgruppe.de/*',
         '*://*.online.bfgruppe.de/*'
```
Comment (AdamWr) [inline compiler/MetadataUtils.ts:366]: It seems that there is a missed comma `,`
The same in this line - https://github.com/AdguardTeam/PopupBlocker/pull/242/files#diff-2ab9c222fff241d52e1a592d1afbe9e017e794544870782a67933fe1e228434dR725

Comment (Alex-302) [reply compiler/MetadataUtils.ts:366]: damn, yes

**Review by zloyden — APPROVED** [review]

File: compiler/MetadataUtils.ts [hunk 2]
```diff
@@ -363,7 +363,10 @@ export default class MetadataUtils {
         '*://*.clashofclans.com/*',
         // https://github.com/AdguardTeam/PopupBlocker/issues/238
         '*://online.bfgruppe.de/*',
-        '*://*.online.bfgruppe.de/*'
+        '*://*.online.bfgruppe.de/*',
+        // https://github.com/AdguardTeam/PopupBlocker/issues/238
```
Comment (AdamWr) [inline compiler/MetadataUtils.ts:367]: I didn't notice it before, but it seems that there is a wrong link to issue.
It should be - `https://github.com/AdguardTeam/PopupBlocker/issues/241`
The same here - https://github.com/AdguardTeam/PopupBlocker/pull/242/files#diff-2ab9c222fff241d52e1a592d1afbe9e017e794544870782a67933fe1e228434dR726

Comment (Alex-302) [reply compiler/MetadataUtils.ts]: fixed both links

**Review by AdamWr — APPROVED** [review]

**Pull request closed without merging**